src/Ocelot/Configuration/File/FileGlobalConfiguration.cs

```diff
@@ -17,6 +17,14 @@ public FileGlobalConfiguration()
 
         public string RequestIdKey { get; set; }
 
+        /// <summary>
+        /// The timeout in seconds for requests.
+        /// </summary>
+        /// <value>
+        /// The timeout value in seconds.
+        /// </value>
+        public int? TimeoutSeconds { get; set; }
```

Comment:
I would suggest using the same property names for the file global configuration and file route (eg. Timeout or TimeoutMs), otherwise it's misleading. Yes, I agree, I have used seconds in the MessageInvoker pool implementation, but it wasn't exposed to the user.

Reply:
`Timeout` is better

Reply:
Disagree to use milliseconds.
Agree to use seconds! Read my [comment](https://github.com/ThreeMammals/Ocelot/pull/2073#discussion_r1622477480)